### PR TITLE
feature(auth):auth sdk add GetListByAccountAsync、GetSecurityTokenAsync

### PR DESCRIPTION
**Pull request by 15168440402**

### CODE REVIEW

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Auth.Contracts/Model/SecurityTokenModel.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    public string Bucket { get; set; }
+
+    public SecurityTokenModel(string region, string accessKeyId, string accessKeySecret, string stsToken, string bucket)
+    {
+        Region = region;
+        AccessKeyId = accessKeyId;
+        AccessKeySecret = accessKeySecret;
+        StsToken = stsToken;
+        Bucket = bucket;
```
Comment (zhenlei520) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Auth.Contracts/Model/SecurityTokenModel.cs:24]: ?

**Review by zhenlei520 — APPROVED** [review]
@doddgu LGTM

**Review by doddgu — APPROVED** [review]
LGTM

**Comment by zhenlei520** [comment]
@doddgu Auth provides SecurityToken?

**Pull request merged**